src/components/Results.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+
+class Results extends Component {
+    render() {
+        return (
+            <div>
+                <table className="table search-table">
+                    <thead>
+                        <tr>
+                            <th>Search Results</th>
```

Comment:
I think this would be better off outside the `<table>`, just in a `<div>` perhaps.